docs/import/manage_disaster.html

```diff
@@ -104,12 +104,12 @@
       <div>
         <table id="asset-selection-table">
           <thead>
-          <tr id="score-asset-header-row">
-          </tr>
+          <tr id="score-asset-header-row"><td>Score Assets</td></tr>
           </thead>
           <tbody id="asset-selection-table-body"></tbody>
         </table>
       </div>
+      <div><label for="damage-asset-select">Damage asset:</label><select id="damage-asset-select"></select>
```

Comment:
nit: add space after 'Damage asset: ' (i think in reality we want padding around all our labels, but understandable if you don't want to futz with that right now)